Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1035,16 +1040,16 @@ HolidayNames: !dictionary
     whiteloverday: [ whiteloverday ] # TODO or remove?
     loverday: [ loverday ] # TODO or remove?
     groundhougday: [ groundhougday, bosmarmottendag ]
-    inaugurationday: [ inaugurationday, inauguratiedag ] 
+    inaugurationday: [ inaugurationday, inauguratiedag ]
     arborday: [ arborday ] # TODO or remove?
```

Comment:
If these holidays marked with TODO are not really mentioned in Dutch, feel free to remove them.

Reply:
updated.